src/main.js

```diff
@@ -65,9 +65,7 @@ window.addEventListener("DOMContentLoaded", () => {
         event.preventDefault();
         const searchInput = document.querySelector('.qld-search-input input');
         const query = searchInput.value.trim();
-        if (query) {
-          submitSearchForm(query);
```

Comment:
Sorry @ghazal-salehi I didn't realise earlier - does this work? as query won't exist unless searchInput has a present value (failing the empty submit scenario) 

I have since changed submitSearchForm(query='', form) where query is empty unless provided